packages/core/src/components/HelpDrawer/HelpDrawer.scss

```diff
@@ -35,16 +34,22 @@
   }
 }
 
-.ds-c-help-drawer__header {
-  position: sticky;
-  top: 0;
+.ds-c-help-drawer__body {
+  flex-grow: 1; // Stretch so that the container takes up the rest of the screen
+  position: relative;
+  width: 100%;
 }
 
-.ds-c-help-drawer__body {
-  // Stretch the body so that the footer sticks to the
-  // bottom of the screen
-  flex: 1 0 auto;
+.ds-c-help-drawer__content-container {
+  bottom: 0;
```

Comment:
I don't think the styles for `ds-c-help-drawer__content-container` are needed I removed them and got the same experience after moving the footer.